packages/experimental/Tabs/src/useTabs.ts

```diff
@@ -1,10 +1,16 @@
 import * as React from 'react';
 import { View } from 'react-native';
 import { useSelectedKey } from '@fluentui-react-native/interactive-hooks';
-import { TabsProps, TabsState } from './Tabs.types';
+import { TabsProps, TabsState, TabsInfo } from './Tabs.types';
 
-export const useTabs = (props: TabsProps): TabsState => {
-  // Attach the pressable state handlers.
+/**
+ * Re-usable hook for FURN Tabs.
+ * This hook configures tabs props and state for FURN Tabs.
+ *
+ * @param props user props sent to FURN Tabs
+ * @returns configured props and state for FURN Tabs
```

Comment:
I don't think we actually use "FURN" anywhere in code. Could we just omit that word?

Reply:
Sure